### PR TITLE
fixed gender color

### PR DESCRIPTION
**Pull request by LeonTeraoka**
Summary
It turns out that the gender color is not reflected because there was no if statement in show.html.haml to handle the condition.

Result
The gender color will be reflected on the screen after posting.

### CODE REVIEW

File: app/views/teams/show.html.haml [hunk 1]
```diff
@@ -2,42 +2,126 @@
...
-
-          .tags
-            .chip
-              = @team.rank_range
-            - @game_types.each do |g|
+        - if @team.gender == 'man'
+          .team-header-man
+            .summoner_name
+              %br/
+              %img.circle.responsive-img{ src: "http://ddragon.leagueoflegends.com/cdn/10.21.1/img/profileicon/#{@team.profile_image}.png", alt: "profile_img"}
```
Comment (rutko) [inline app/views/teams/show.html.haml:9]: The current version of ddragon is 10.23.1 

https://developer.riotgames.com/docs/lol

**Review by rutko — CHANGES_REQUESTED** [review]
Please fix comment.

**Comment by LeonTeraoka** [comment]
Thanks for your confirmation.
It's easy to fix it but please give me time because of study.

By the way, your comment "Please fix comment." 
It sounds like my comment is incorrect for the changes.
You can rather say like "Please fix according to my comment"

**Comment by rutko** [comment]
OK.
I understand that it will take time 😌
And i understood "Please fix according to my comment" 👍

**Pull request merged**